doc: Describe CAF module suspend/resume request events support (#23556)
nrfconnect/sdk-nrf: doc/nrf/libraries/caf/ble_adv.rst

With the :kconfig:option:`CONFIG_BT_PRIVACY` Kconfig option enabled, set the :kconfig:option:`CONFIG_CAF_BLE_ADV_ROTATE_RPA` option to synchronize Resolvable Private Address (RPA) rotation with the undirected advertising data update.
You can control the rotation period with the :kconfig:option:`CONFIG_CAF_BLE_ADV_ROTATE_RPA_TIMEOUT` option and change the randomization factor of the rotation period with the :kconfig:option:`CONFIG_CAF_BLE_ADV_ROTATE_RPA_TIMEOUT_RAND` option.

Suspending the module
=====================

> When the :kconfig:option:`CONFIG_CAF_BLE_ADV_MODULE_SUSPEND_EVENTS` Kconfig option is enabled, the module can be suspended using module suspend request event (:c:struct:`module_suspend_req_event`) directed to this module.

Reviewer: ```suggestion
When the :kconfig:option:`CONFIG_CAF_BLE_ADV_MODULE_SUSPEND_EVENTS` Kconfig option is enabled, you can suspend the module using a module suspend request event (:c:struct:`module_suspend_req_event`) directed to this module.
```

When entering the suspended state, the module stops Bluetooth LE advertising and disconnects connected peers.

> The module can be resumed using module resume request event (:c:struct:`module_resume_req_event`) directed to this module.

Reviewer: ```suggestion
You can resume the module using a module resume request event (:c:struct:`module_resume_req_event`) directed to this module.
```
